plugins/upload/trumbowyg.upload.js

```diff
@@ -58,6 +58,11 @@
                 file: 'Fil',
                 uploadError: 'Fejl'
             },
+	    de: {
+		upload: 'Hochladen',
+		file: 'Datei',
+		uploadError: 'Fehler'
+	    },
```

Comment:
indent

Reply:
My IDE shows both versions alike, but here's for you vi guys :)